database/database.js

```diff
@@ -1,22 +1,58 @@
 const mongoose = require('mongoose');
 
+// Actually connecting to the database
 mongoose.connect('mongodb://localhost:27017/sidebar?gssapiServiceName=mongodb', {useNewUrlParser: true, useUnifiedTopology: true})
 .then(() => console.log("Connected to database."))
 .catch( error => console.log("Connection error: ", error));
 
 const connection = mongoose.connection;
 
+// ------------------------------------------------------------------
+// Setting up schemas and models
 const priceSchema = new mongoose.Schema({
   courseID: Number,
   basePrice: Number,
   discountPercentage: Number,
   discountedPrice: Number,
   saleEndDate: Date,
   saleOngoing: Boolean
-})
+});
 
 const Price = mongoose.model('Price', priceSchema);
 
+const previewVideoSchema = new mongoose.Schema({
+  courseID: Number,
+  previewVideoUrl: String,
+});
+
+const PreviewVideo = mongoose.model('PreviewVideo', previewVideoSchema);
+
+const sidebarSchema = new mongoose.Schema({
+  courseID: Number,
+  fullLifetimeAccess: String,
+  accessTypes: String,
+  assignments: Boolean,
+  certificateOfCompletion: Boolean
+})
+
+const Sidebar = mongoose.model('Sidebar', sidebarSchema);
+
+// ------------------------------------------------------------------
+// Functions that retrieve information from the database
+const getPrice = (query, callback) => {
+  return Price.find(query, callback);
+}
+
+const getPreviewVideo = (query, callback) => {
+  return PreviewVideo.find(query, callback);
+}
+
+const getSidebar = (query, callback) => {
+  return Sidebar.find(query, callback);
+}
+
+// ------------------------------------------------------------------
+// Functions that create data and add it to the database
```

Comment:
Logic looks good here, I would consider refactoring your database seeding function into another module

Reply:
I was thinking about that. Probably not a bad idea.